src/Illuminate/Support/Str.php

```diff
@@ -635,12 +635,19 @@ public static function words($value, $words = 100, $end = '...')
      *
      * @param  string  $string
      * @param  array  $options
+     * @param  array  $extensions
      * @return string
      */
-    public static function markdown($string, array $options = [])
+    public static function markdown($string, array $options = [], array $extensions = [])
     {
         $converter = new GithubFlavoredMarkdownConverter($options);
 
+        $environment = $converter->getEnvironment();
+
+        foreach ($extensions as $extension) {
+            $environment->addExtension($extension);
+        }
```

Comment:
The environment is only needed when `$extensions !== []`. Wouldn't it make sense, to wrap this in an `if`?